src/main/java/fr/cnes/sonarqube/plugins/framac/sensor/FramaCSensor.java

```diff
@@ -61,6 +65,23 @@ public void describe(final SensorDescriptor sensorDescriptor) {
         sensorDescriptor.createIssuesForRuleRepositories(FramaCRulesDefinition.getRepositoryKeyForLanguage());
     }
 
+    /**
+     * Check if a file is a CSV file by analysing the header (first line)
+     * @param filePath : the file path
+     * @return true if the file is a csv file, false otherwise
+     */
+    private boolean isCsvFile(String filePath) {
+        boolean isCsvFile = false;
+        try (BufferedReader br =
+            new BufferedReader(new FileReader(filePath))) {
+            String line = br.readLine();
+            if (line.startsWith("directory\tfile")) {
+                isCsvFile = true;
+            }
+        } catch (IOException e) {}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either remove or fill this block of code. ([squid:S00108](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AS00108))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonar-frama-c-plugin&pullRequest=23&issues=AWY0GieMM63is4RD-5M5&open=AWY0GieMM63is4RD-5M5)

[ISSUE_KEY]: AWY0GieMM63is4RD-5M5